tools/go_branch.sh

```diff
@@ -49,7 +49,7 @@ declare -r go_branch=$(git show-ref --hash go)
 # current go_branch directory. We move to the new repository for convenience.
 declare -r repo_orig="$(pwd)"
 declare -r repo_new="${tmp_dir}/repository"
-git clone . "${repo_new}"
+git clone --depth 1 . "${repo_new}"
```

Comment:
The other ones are fine, but this one requires the full history.

Reply:
@amscanne fixed